osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -319,28 +284,80 @@ private void load(LocalisationManager localisation)
                     }
                 };
 
+                addInfoLabels();
+
                 titleBinding.BindValueChanged(_ => setMetadata(metadata.Source));
                 artistBinding.BindValueChanged(_ => setMetadata(metadata.Source), true);
+                starDifficulty.BindValueChanged(updateDifficulty, true);
 
                 // no difficulty means it can't have a status to show
                 if (beatmapInfo.Version == null)
                     StatusPill.Hide();
+            }
 
-                addInfoLabels();
+            private void updateDifficulty(ValueChangedEvent<StarDifficulty?> valueChanged)
```

Comment:
i'd hope this is restored too

Reply:
Yes `addInfoLabels` is further above (https://github.com/ppy/osu/pull/12503/files#diff-10508e952755970ee0b1dfc1cfaac15497e2e8153598c73c0133e3a3e367ca5fR287)

Reply:
I mean removing all this star difficulty update logic. It should be in a separate PR.

This PR should *only do what it says in the title*. Try reading the diff yourself. It's legit impossible to parse right now.

Reply:
Yes but the wedge shouldn't loose functionality. I somewhat need to recreate the StarDifficulty and the bar otherwise they won't be updatet anymore on a mod change. On the other hand i also can recreate the whole info layer on a mod change if you would prefer that for this pr